integration-tests/src/test/scala/za/co/absa/spline/harvester/SparkUnimplementedCommandsSpec.scala

```diff
@@ -0,0 +1,285 @@
+/*
+ * Copyright 2019 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.spline.harvester
+
+import java.util.Properties
+
+import org.apache.spark.sql.types.{IntegerType, StringType, StructField, StructType}
+import org.apache.spark.sql.{DataFrame, Row, SparkSession}
+import org.scalatest.{FlatSpec, Ignore, Matchers}
+import za.co.absa.spline.common.TempDirectory
+import za.co.absa.spline.common.Version.VersionOrdering.{max => _, min => _}
+import za.co.absa.spline.test.fixture.spline.SplineFixture
+import za.co.absa.spline.test.fixture.{JDBCFixture, SparkDatabaseFixture, SparkFixture}
+
+/**
+ * Tests in this class serve as a way to produce unimplemented spark commands.
+ * They can be used as a template once the implementation begins.
+ *
+ * None of the tests is supposed to pass yet and therefore they are ignored.
+ *
+ */
+@Ignore
+class SparkUnimplementedCommandsSpec extends FlatSpec
+  with Matchers
+  with SparkFixture
+  with SplineFixture
+  with SparkDatabaseFixture
+  with JDBCFixture {
+
+  val databaseName = "testDb"
+  val tableName = "testTable"
+
+  private val tempDirPath = TempDirectory(prefix = "test").deleteOnExit().path.toFile.getAbsolutePath
+
+  "Lineage for create database" should "be caught" in
+    withCustomSparkSession(_
+      .enableHiveSupport()
+      .config("hive.exec.dynamic.partition.mode", "nonstrict")) { spark =>
+        try
+          withLineageTracking(spark) { lineageCaptor =>
+            val (plan, _) = lineageCaptor.lineageOf {
+              spark.sql(s"CREATE DATABASE $databaseName") // CreateDatabaseCommand
+            }
+          }
+        finally
+          spark.sql(s"DROP DATABASE $databaseName CASCADE")
```

Comment:
Usually a test method shouldn't be responsible for cleanup, to keep the tests more concise and focused. The cleanup should rather be done in one the fixtures or `before/after` methods.

In this particular case you can use `withRestartingSparkContext` loan-method that would restart Spark and do all the cleanup for you.